packages/sanity/src/core/comments/components/mentions/MentionsMenu.tsx

```diff
@@ -25,23 +20,39 @@ const FlexWrap = styled(Flex)({
   maxHeight: ITEM_HEIGHT * MAX_ITEMS + LIST_PADDING * 2 + ITEM_HEIGHT / 2,
 })
 
+export interface MentionsMenuHandle {
+  selectCurrent: () => void
+  setSearchTerm: (term: string) => void
+}
 interface MentionsMenuProps {
   loading: boolean
+  inputElement?: HTMLDivElement | null
   onSelect: (userId: string) => void
   options: MentionOptionUser[] | null
 }
 
 export const MentionsMenu = React.forwardRef(function MentionsMenu(
   props: MentionsMenuProps,
-  ref: React.Ref<HTMLDivElement>,
+  ref: React.Ref<FIXME>,
```

Comment:
I think this can be changed to: `React.Ref<MentionsMenuHandle>` and the `ref={ref}` added to the `Flex` which will get an TS error can be removed (it is not being used for anything)

Reply:
Yes, well spotted! I've force-pushed a change now that does exactly this.